src/features/nav/NavCurrentOrg.tsx

```diff
@@ -27,7 +28,7 @@ export const NavCurrentOrg = ({ org }: { org: NavOrg }) => {
         to={paths.vaultsForOrg(org.id)}
         icon={<DollarSign />}
       />
-      {isInOrg && (
+      {isFeatureEnabled('org_view') && isInOrg && (
```

Comment:
Does this really work tho? where is the code that says "only show this if there are members" is it the isInOrg check?